public/app/features/dashboard-scene/scene/layout-tabs/TabItems.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from 'react';
+
+import { OptionsPaneCategoryDescriptor } from 'app/features/dashboard/components/PanelEditor/OptionsPaneCategoryDescriptor';
+
+import { MultiSelectedEditableDashboardElement } from '../types/MultiSelectedEditableDashboardElement';
+
+import { TabItem } from './TabItem';
+import { getEditOptions, renderActions } from './TabItemsEditor';
+
+export class TabItems implements MultiSelectedEditableDashboardElement {
+  public readonly isMultiSelectedEditableDashboardElement = true;
+  public readonly typeName = 'Tabs';
+
+  public constructor(private _tabs: TabItem[]) {}
+
+  public useEditPaneOptions(): OptionsPaneCategoryDescriptor[] {
+    return getEditOptions(this);
+  }
+
+  public renderActions(): ReactNode {
+    return renderActions(this);
+  }
+
+  public getTabs(): TabItem[] {
+    return this._tabs;
+  }
+
+  public onDelete() {
+    this._tabs.forEach((tab) => tab.onDelete());
```

Comment:
Tab deletion does not work for either single or multi selected tabs, not sure if I'm doing something wrong here.

Reply:
nvm I was using it wrong 😆